### PR TITLE
end my miserable life

### PR DESCRIPTION
**Pull request by PKPenguin321**
https://www.youtube.com/watch?v=PTAXUYLbFYk

:cl: PKPenguin321
tweak: Sand now fits in your pockets
rscadd: Sand can now be thrown into people's eyes, doing slight stamina damage, making their vision slightly blurry, and making them stumble when they walk for a short time.
/:cl:

### CODE REVIEW

**Comment by KazeEspada** [comment]
I did this years ago, but I kept bundling it with a screwdriver nerf. So it kept getting denied.

**Comment by PKPenguin321** [comment]
also this is untested and has no changelog as of yet so dont merge it too soon

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -60,6 +61,20 @@
 	qdel(src)
 	return
 
+/obj/item/weapon/ore/glass/throw_impact(atom/hit_atom)
+	if(..() || !iscarbon(hit_atom))
+		return
+	var/mob/living/carbon/C = hit_atom
+	if((C.head && C.head.flags_cover & HEADCOVERSEYES) || \
+		(C.wear_mask && C.wear_mask.flags_cover & MASKCOVERSEYES))
+		visible_message("<span class='danger'>[C]'s mask/helmet blocks the sand!</span>")//glasses are not thick enough to stop it
```
Comment (Cheridan) [inline code/modules/mining/ores_coins.dm]: >mask/helmet

lazy

Comment (Niknakflak) [inline code/modules/mining/ores_coins.dm]: >glasses are not thick enough to stop it

gimme a break

Comment (KorPhaeron) [inline code/modules/mining/ores_coins.dm]: That fucking visible message

Comment (PKPenguin321) [inline code/modules/mining/ores_coins.dm]: Supposedly it'll pick the right one or some shit but like I already said this PR is not merge ready, I'm gonna touch this stuff up probably tomorrow. No bully

**Comment by PKPenguin321** [comment]
okay, i've added a changelog, fixed to the code so that it isn't absolute garbage, and actually tested it in-game. this is good for review/merge

**Comment by optimumtact** [comment]
I give this a merge conflict/10

:+1:

**Comment by PKPenguin321** [comment]
ahaha i'm gonna kill myself

File: code/modules/mining/ores_coins.dm (hunk 1 above)
Comment (Jordie0608) [inline code/modules/mining/ores_coins.dm]: You would need to insert the name of the mask or helmet as a [var] to get it to use the correct name; 'make/helmet' does nothing

**Comment by PKPenguin321** [comment]
FUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUCKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK

**Pull request closed without merging**